iroh-sync/src/net/codec.rs

```diff
@@ -459,10 +498,12 @@ mod tests {
         let (mut bob_reader, mut bob_writer) = tokio::io::split(bob);
         let bob_store = bob_store.clone();
         let bob_task = tokio::task::spawn(async move {
-            run_bob::<S, _, _>(
+            run_bob::<S, _, _, _, _>(
                 &mut bob_writer,
                 &mut bob_reader,
-                bob_store,
+                |namespace, _| {
+                    futures::future::ready(bob_store.open_replica(&namespace).map(Into::into))
```

Comment:
same here

Reply:
deferred to https://github.com/n0-computer/iroh/issues/1502